stock_putaway_storage_type_abc/models/stock_location.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+
+
+ABC_SELECTION = [('a', 'A'), ('b', 'B'), ('c', 'C')]
+
+
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    pack_storage_strategy = fields.Selection(
+        selection_add=[('abc', 'Chaotic ABC')]
+    )
+    display_abc_storage = fields.Boolean(
+        compute='_compute_display_abc_storage'
+    )
+    abc_storage = fields.Selection(ABC_SELECTION, required=True, default='b')
+
+    @api.depends('location_id', 'location_id.pack_storage_strategy')
```

Comment:
I think this will invalidate only for one level (well, shouldn't be too much of an issue as long as it's not stored).
Couldn't you do the same check with `location_id.display_abc_storage`  beside `location_id.pack_storage_strategy`? If the parent displays ABC, so the children.